src/test.ts

```diff
@@ -176,6 +176,15 @@ test('can create a two-phase transfer', async (): Promise<void> => {
   assert.strictEqual(accounts[1].credits_reserved, 0n)
   assert.strictEqual(accounts[1].debits_accepted, 100n)
   assert.strictEqual(accounts[1].debits_reserved, 50n)
+
+  //Lookup the transfer
+  const transfers = await client.lookupTransfers([transfer.id])
+  assert.strictEqual(transfers.length, 1)
```

Comment:
Shall we reorder these checks according to the struct field order? Also see if there any more fields we can assert on? Even if 0?

Reply:
I agree. Done this. :-)